.github/workflows/generate-docs.yaml

```diff
@@ -0,0 +1,54 @@
+name: Generate Documentation
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Generate documentation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+      - uses: actions/setup-go@v1
+        with:
+          go-version: '1.16'
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.9
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: docs-pip-${{ runner.os }}-${{ hashFiles('doc-requirements.txt') }}-${{ hashFiles('doc-requirements.in') }}
+          restore-keys: docs-pip-
+      - name: Install Dependencies
+        run: |
+          # Install all requirments
+          make doc-requirements.txt
```

Comment:
This doesn't actually install the requirements, it updates the file with latest values of requirement.
you will need to run `pip install -r doc-requirements.txt` to install...

Reply:
Thanks